readme-vars.yml

```diff
@@ -2,7 +2,7 @@
 
 # project information
 project_name: piper
-project_url: "https://github.com/rhasspy/piper/"
+project_url: "https://github.com/rhasspy/wyoming-piper"
```

Comment:
is this correct? when you go to the old url, it says `Development has moved: https://github.com/OHF-Voice/piper1-gpl`